.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended", // uses the recommended rules from the @typescript-eslint/eslint-plugin
+    "plugin:jest/recommended" // enables eslint-plugin-jest
+  ],
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "ignorePatterns": [
+    "bin/",
+    "lib/"
+  ],
+  "rules": {
+    "quotes": ["error", "double"],
+    "indent": ["error", 4, { "SwitchCase": 1 }],
```

Comment:
HAPNodeJS uses two, not sure if we want to follow in style.

Reply:
That is exactly one of the things I wanted to be discussed with my initial commit.

HAP-NodeJS is basically using both 😅 It's not quite consistent.
I personally grew up using four so that's where the default values is coming from. But idk, also long we can be consistent on one.